tests/propagation/test_http.py

```diff
@@ -0,0 +1,49 @@
+from unittest import TestCase
+from nose.tools import eq_, ok_
+from tests.test_tracer import get_dummy_tracer
+
+from ddtrace.span import Span
+from ddtrace.context import Context, ThreadLocalContext
+
+from ddtrace.propagation.http import (
+    HTTPPropagator,
+    HTTP_HEADER_TRACE_ID,
+    HTTP_HEADER_PARENT_ID,
+    HTTP_HEADER_SAMPLING_PRIORITY,
+)
+
+class TestHttpPropagation(TestCase):
+    """
+    Tests related to the ``Context`` class that hosts the trace for the
+    current execution flow.
+    """
+    def test_inject(self):
+        tracer = get_dummy_tracer()
+
+        with tracer.trace("global_root_span") as span:
+            headers = {}
+            propagator = HTTPPropagator()
+            propagator.inject(span.context, headers)
+
+            eq_(int(headers[HTTP_HEADER_TRACE_ID]), span.trace_id)
+            eq_(int(headers[HTTP_HEADER_PARENT_ID]), span.span_id)
+            # TODO: do it for priority too
```

Comment:
missing something here?

Reply:
I'll add the priority in this test as part of my next PR to change the way we set/get the priority.